### PR TITLE
Setup [24][28][29][33]

### LINKED ISSUE #24: Icon component
Add SVG Icon component to the project 
https://github.com/MMF-FE/vue-svgicon

### LINKED ISSUE #28: Base Layout
That has Header, Body and Footer @ all with tailwindcss classes

### LINKED ISSUE #29: Add Gotham Pro font

### LINKED ISSUE #31: Update vuex-bitshares dependencies
Some vuex-bitshares dependencies are outdated which sometimes causes crash when cloning project

### LINKED ISSUE #32: Make serveral issues templates
For feature/bug etc 

https://blog.github.com/2018-05-02-issue-template-improvements/

### LINKED ISSUE #33: Add unit tests to pre-push hook
Prevents pushing code that doesn't pass tests

### PR DESCRIPTION
**Pull request by roma219**
#### What's this PR do?
Makes a base layout aka Header/Body/Footer, adds svg icons components, adds fonts, adds template for new issues, adds running unit-tests as a pre-push hook
#### Where should the reviewer start?
`docs/dev-guide.md` for info on icons, `src/router.js` for base layout structure understanding, `src/styles/main.scss` for styles structure understanding
#### How should this be manually tested?
Try adding some svg as an icon component, use some new fonts, navigate to /components and /auth routes.
#### Any background context you want to provide?
n/a
#### What are the relevant tickets?
closes #24 
closes #28 
closes #29 
closes #33 
#### Screenshots (if appropriate)
image

#### Questions:
n/a

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,6 +5,8 @@ node_modules
 /tests/e2e/reports/
 selenium-debug.log
 
+src/assets/icons/
```
Comment (youaresofunny) [inline .gitignore:8]: So we'll need to regenerate svg's at each pull? Mb keep it?

File: docs/dev-guide.md [hunk 2]
```diff
@@ -72,4 +72,19 @@ Please follow the vue official guidilines for naming and structuring components
 To run styleguide use command `npm run styleguide` .     
 [How to document components](https://github.com/vue-styleguidist/vue-styleguidist/blob/master/docs/Documenting.md)
 
+## Icons
+
+#### Adding new
```
Comment (youaresofunny) [inline docs/dev-guide.md]: I think we can add a bit more info here (where generated images stored)

File: .gitignore (hunk 1 above)
Comment (roma219) [reply .gitignore:8]: The problem is that when you run `npm run svg` all icons are generated and git recognizes them as new so you got to add and commit them. Insted we add this folder to `gitignore`, keep actual svg files in git (`src/assets/svg`) and this way we'll only need to generate them when there were some new icons added - won't happen very often as most of the icons are gonna be provided & generated soon.

File: docs/dev-guide.md (hunk 2 above)
Comment (roma219) [reply docs/dev-guide.md]: Yeah, i'll add this info here

**Comment by roma219** [comment]
@youaresofunny done

**Review by youaresofunny — APPROVED** [review]

**Review by hisinbox — APPROVED** [review]
clean and clear. may amend going forward based on examples

**Pull request merged**